src/Queue/RabbitMQQueue.php

```diff
@@ -577,7 +577,7 @@ protected function getRandomId(): string
      */
     public function close(): void
     {
-        if (! $this->currentJob->isDeletedOrReleased()) {
+        if ($this->currentJob && ! $this->currentJob->isDeletedOrReleased()) {
```

Comment:
```suggestion
        if (isset($this->currentJob) && ! $this->currentJob->isDeletedOrReleased()) {
```

This should have the same effect